.github/scripts/update-pr-with-issues.js

```diff
@@ -0,0 +1,178 @@
+const https = require('https');
+
+function containsIssueReference(text, issueNumber) {
+  const regex = new RegExp(`(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved):?\\s*#?${issueNumber}\\b`, 'i');
+  return regex.test(text);
+}
```

Comment:
- this will catch issues referenced like this`closes #123` 
- but not `closes https://github.com/metabase/metabase/issues/123`

there's a better regex in `linked-issues.ts` that you can probably adapt